src/main/java/org/commcare/formplayer/session/MenuSession.java

```diff
@@ -151,14 +151,14 @@ public NotificationMessage updateApp(String updateMode) {
         }
     }
 
-    private void resolveInstallReference(String installReference, String appId, String host){
+    private static String resolveInstallReference(String installReference, String appId, String host, String domain){
```

Comment:
changed this and `getReferenceToLatest` to pure, static functions rather than mutating state on the class